src_types/common/anyString$wSelector.d.ts

```diff
@@ -0,0 +1,4 @@
+declare namespace $w {
+    type $w = (selector: string) => any
+}
+declare function $w(selector: string): any;
```

Comment:
did you find out why do we need to define both `$w` & `$w.$w` ?

Reply:
yes, there are some event handlers in declarations.d.ts that use $w.$w as a parameter